### DISCUSSION TITLE
How to show bubble menu within a table?

### DISCUSSION THREAD

**Discussion by dkrusenstrahle** [ueberdosis/tiptap#1268] [General Questions]
Hello,

I am developing an Tiptap 2.0 app.
I am using Tables and Bubble menu in my editor.

Is it possible to get the bubble menu to appear when focusing on a table or one of it's cells?
I would like to use that to show tools for the table.

Thank you!

**Comment by philippkuehn** [comment]
With [this patch](https://github.com/ueberdosis/tiptap/commit/6472d2c2715bd29a061abae6a59963949c298e55) it should work better now.

**Reply by TooBug** [reply]
I am facing the same problem. It's still not possible to show a bubble menu when **FOCUSING** on a table.

This patch may handled the selection in tables, not the focusing thing.

**Reply by bryanjtc** [reply]
Did you found a solution?

**Comment by hjerichen** [comment]
You can simply override the shouldShow logic:
https://tiptap.dev/docs/editor/api/extensions/bubble-menu#should-show

**Comment by jessehanley** [comment]
```js
shouldShow: ({ editor, view, state, oldState, from, to }) => {
        return editor.isActive('table') || editor.isActive('tableCell') || editor.isActive('tableHeader') || editor.isActive('tableRow');
    },
```
^This works for others looking at this thread.
